server/lomas_server/dp_queries/dummy_dataset.py

```diff
@@ -66,6 +67,10 @@ def make_dummy_dataset(  # pylint: disable=too-many-locals
                     ),
                     dtype=np.dtype(dtype),
                 )
+            case IntCategoricalMetadata():
+                dtype = f"{data.type}{data.precision}"
+                categories = data.categories
+                serie = pd.Series(rng.choice(categories, size=nb_rows), dtype=np.dtype(dtype))
```

Comment:
I think you have to rename categories to something else (`int_categories`) otherwise suffer mypy wrath due to line 51 being another type ^^